### ISSUE TITLE
dumb question: how do you save?

### ISSUE THREAD

**Issue by ck-on** [kud/firefox-simple-text-editor#14]
If I use File -> Save Page As, I simply get a stub file with the ace editor header.

Can't figure out how to save for the life of me.

**Comment by kud** [comment]
Indeed, it could be great to be able to save. I'll see what I could do. Thanks ;)

**Comment by ck-on** [comment]
Ah okay so I am not crazy then, thought I was doing something wrong or missing a button.

It looks like the editor creates a frame within the page, perhaps it is possible to trick the "save page as" in firefox to save the innerHtml of the frame instead of the outside container?

Thanks for trying to do it!
